bench/scrypt.cc

```
@@ -0,0 +1,46 @@
+// limitations under the License.
+
+#include <benchmark/benchmark.h>
+
+#include <openssl/base.h>
+#include <openssl/err.h>
+#include <openssl/evp.h>
+
+#include "./internal.h"
+
+namespace {
+const char kPassword[] = "password";
```

Clang (14.0.6)

[warning] do not declare C-style arrays, use std::array<> instead

(modernize-avoid-c-arrays)